xml/storage_multipath.xml

```diff
@@ -1131,393 +1183,300 @@
+    </para>
+    <note>
+      <title>Disabling invocation of kpartx</title>
+      <para>
+	The <literal>skip_kpartx</literal> option in
+	<filename>/etc/multipath.conf</filename> can be used to disable
+	invocation of <command>kpartx</command> on selected multipath maps. This
+	may be useful on virtualization hosts, for example.
+      </para>
+    </note>
+    <para>
+      Partition tables and partitions on multipath devices can be manipulated
+      as usual, using &yast; or tools like <command>fdisk</command> or <command>parted</command>. Changes
+      applied to the partition table will be noted by the system when the
+      partitioning tool exits. If this doesn't work (usually because a device
+      is busy), try <command>multipathd reconfigure</command>, or reboot the
+      system.
+    </para>
+    <para>
+      A partitioned multipath device can't be used otherwise. For example, you
+      can't create an LVM physical volume from a partitioned device. You will
+      need to wipe the partition table before doing this.
+    </para>
   </sect2>
  </sect1>
  <sect1 xml:id="sec-multipath-conf-file">
-  <title>Creating or modifying the /etc/multipath.conf file</title>
+  <title>Multipath configuration</title>
 
   <para>
-   The <filename>/etc/multipath.conf</filename> file does not exist unless you
-   create it. Default multipath device settings are applied automatically when
-   the <command>multipathd</command> daemon runs unless you create the
-   multipath configuration file and personalize the settings.
+    The built-in <systemitem>multipath-tools</systemitem> defaults work well for most
+    setups. If customizations are needed, a configuration file needs to be
+    created. The main configuration file is
+    <filename>/etc/multipath.conf</filename>. In addition, files matching the pattern
+    <filename>/etc/multipath/conf.d/*.conf</filename> are read in alphabetical
+    order. See   <xref linkend="sec-multipath-conf-file-syntax"
+    xrefstyle="SectTitleOnPage"/> for precedence rules.
   </para>
+  <note>
+    <title>
+      Generated configuration files
+    </title>
+    <para>
+      The files <filename>/etc/multipath/wwids</filename>,
+      <filename>/etc/multipath/bindings</filename>, and
+      <filename>/etc/multipath/prkeys</filename> are maintained by
+      <systemitem>multipath-tools</systemitem> to store persistent information
+      about previously created multipath maps, map names, and reservation keys
+      for SCSI persistent reservations, respectively. They are
+      &ldquo;configuration files&rdquo; in a broader sense, but not meant to
+      be manually edited.
+    </para>
+  </note>
+  <note>
+    <title>Configurable paths</title>
+    <para>
+      Except for <filename>/etc/multipath.conf</filename>, the paths of
+      the configuration directories and files are configurable,
+      but this is discouraged. It will be disallowed in future versions
```

Comment:
I'd omit the information now. Now it is possible to configure the paths. If it is not recommended due to another reason, please state the reason. Changes for future releases are a matter of next releases documentation :)

Reply:
I would like to mention that this is discouraged.

Reply:
I removed the part about the future change.